services/analyzer/mysql-table-constraints-getter.js

```diff
@@ -0,0 +1,54 @@
+function MysqlTableConstraintsGetter(databaseConnection) {
+  const queryInterface = databaseConnection.getQueryInterface();
+
+  this.convertToUniqueIndexArray = (constraints) => {
+    const uniqueIndexes = {};
+    constraints.filter((constraint) => constraint.columnType === 'UNIQUE')
+      .forEach((item) => {
```

Comment:
The data we are working on there are quite hard to follow. May I suggest this naming instead for readability ?
```suggestion
      .forEach((uniqueConstraint) => {
```